pom.xml

```diff
@@ -420,7 +420,6 @@
         <dependency>
             <groupId>org.jenkins-ci.main</groupId>
             <artifactId>jenkins-test-harness</artifactId>
-            <version>2.44</version>
             <scope>test</scope>
```

Comment:
The *jenkins-test-harness* version seems to have been removed here.

Reply:
Well, I though removing the specific version would allow to use the latest test harness.
Added it back.